satpy/readers/insat3d_img_l1b_h5.py

```diff
@@ -179,7 +179,10 @@ def get_area_def(self, ds_id):
         #fov = self.datatree.attrs["Field_of_View(degrees)"]
         fov = 18
         cfac = 2 ** 16 / (fov / cols)
-        lfac = 2 ** 16 / (fov / lines)
+
+        # From reverse engineering metadata from a netcdf file, we discovered
+        # the lfac is actually the same as cfac, ie dependent on cols, not lines!
+        lfac = 2 ** 16 / (fov / cols)
```

Comment:
> the lfac is actually the same as cfac, ie dependent on cols, not lines!

How? Wait. What? But...huh?

Reply:
I kind of makes sense, because this data does not have the same number of columns and lines for the full disc. the cfac and lfac are actually related to the field of view per pixel, and it would be unusual for a geostationary satellite to have non "square" pixels. So the fov per pixel is the same for here in x and y...
Also, to be fair, this is undocumented, as the level 1b data is only provided with longitudes and latitudes for each pixel. So we're probably not supposed to use an area_def for insat data...